client/src/components/CitizenRequest/CitizenRequest.component.tsx

```diff
@@ -1,16 +1,45 @@
-import React from 'react'
-import { Text, Button, Flex } from '@chakra-ui/react'
+import { Text, Button, Flex, useDisclosure } from '@chakra-ui/react'
+import { EmailModal } from '../EmailModal/EmailModal.component'
+import { Agency } from '../../services/AgencyService'
 
 interface CitizenRequestProps {
-  email?: string
-  longName?: string
+  agency?: Agency
+}
+
+// TODO: combine interface Enquiry from both client and server
+export interface Enquiry {
+  questionTitle: string
+  description: string
+  senderEmail: string
 }
 
 const CitizenRequest = ({
-  email = 'askgov@open.gov.sg',
-  longName = '',
+  agency = {
+    id: '',
+    email: 'askgov@open.gov.sg',
+    shortname: 'AskGov',
+    longname: 'AskGov',
+    logo: '',
+  },
```

Comment:
This may come back to haunt us, since it will make it awkward to reason about agency later on, especially since right now, we derive this from the presence of agency information in the URL. I would suggest not having a default here, and instead consider the following:

- HomePage - if not bound to an agency (ie, if rendering the government-wide page), should we allow enquiries to be made at all? If so, what would be an appropriate choice of agency to inject into CitizenRequest?
- Post - related to #23 - if navigating from the government-wide page, can we perhaps infer the agency from the Post agency tags?

@mantariksh - significant proposal made by me, in case I have blind spots.

Reply:
My two cents are that users should have a centralised enquiry place if they do not know which agency to ask, since that is part of what AskGov hopes to aid with.